### DISCUSSION TITLE
Animation priority in v11?

### DISCUSSION THREAD

**Discussion by mixonomicon** [AvaloniaUI/Avalonia#12586] [Q&A]
Hi,

is this priority still used in v11? I am playing with selector triggered animations, and they don't seem to follow the docs. The post-animation value (in FillMode=Forwards) in Diagnostic screen is shown as `LocalValue`. I did not succeed in getting an Animated priority to show up in Diagnostic screen.

**Comment by maxkatz6** [comment]
Yes, animation is still a higher priority than LocalValue https://github.com/AvaloniaUI/Avalonia/blob/c59e63ef6f17bdd73116df8ce177de747f894510/src/Avalonia.Base/Data/BindingPriority.cs#L19

Not sure about devtools though, if they work correctly.
